Add SBOM export documentation to SCA and Library Inventory pages (#36480)
DataDog/documentation: content/en/security/code_security/software_composition_analysis/_index.md

To learn more about how the inventory is generated, how Static and Runtime data differ, and how to interpret the library details (usage, vulnerabilities, licenses, versions, and OpenSSF score), see [Library Inventory][14].

> ### Export a Software Bill of Materials (SBOM)

Reviewer: ```suggestion
### Export a Software Bill of Materials
```

> Export a Software Bill of Materials (SBOM) of your third-party libraries directly from the [Library Inventory][8]. The exported SBOM includes libraries detected both statically (via Static SCA) and at runtime (via Runtime SCA), giving you a single, comprehensive view of your software supply chain.

Reviewer: ```suggestion
Export a SBOM of your third-party libraries directly from the [Library Inventory][8]. The exported SBOM includes libraries detected both statically (with Static SCA) and at runtime (with Runtime SCA), giving you a single, comprehensive view of your software supply chain.
```
